content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Adding Windows worker nodes
+content_type: task
+weight: 50
+---
+
+<!-- overview -->
+
+This page explains how to add Windows worker nodes to a kubeadm cluster.
+
+## {{% heading "prerequisites" %}}
+
+* A running [Windows Server 2022](https://www.microsoft.com/cloud-platform/windows-server-pricing)
+(or higher) instance with administrative access.
+* A running kubeadm cluster created by `kubeadm init` and following the steps
+in the document [Creating a cluster with kubeadm](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
+
+<!-- steps -->
+
+## Adding Windows worker nodes
+
+{{< note >}}
+To facilitate the addition of Windows worker nodes to a cluster, PowerShell scripts from the repository
+https://sigs.k8s.io/sig-windows-tools are used.
+{{< /note >}}
+
+Do the following for each machine:
+
+1. Connect to the machine by using SSH or another method.
+1. Make sure you are Administrator or a privileged user.
+1. Open a PowerShell session.
```

Comment:
(nit) You can [run PowerShell locally](https://learn.microsoft.com/en-us/powershell/scripting/learn/ps101/08-powershell-remoting) and invoke the commands remotely.

Reply:
reworded to tell the user to "open powershell session on the machine"